notNeededPackages.json

```diff
@@ -1080,6 +1080,12 @@
             "sourceRepoURL": "https://github.com/Realytics/fork-ts-checker-webpack-plugin",
             "asOfVersion": "0.4.5"
         },
+        {
+            "libraryName": "form-data",
+            "typingsPackageName": "form-data",
+            "sourceRepoURL": "https://github.com/form-data/form-data",
+            "asOfVersion": "2.5.0"
+        },
```

Comment:
This already exists in line 1072.

Reply:
ops...Didn't notice that. I'll delete this.